container/go/cmd/pusher/pusher.go

```diff
@@ -44,6 +44,7 @@ var (
 	format              = flag.String("format", "", "The format of the uploaded image (Docker or OCI).")
 	clientConfigDir     = flag.String("client-config-dir", "", "The path to the directory where the client configuration files are located. Overiddes the value from DOCKER_CONFIG.")
 	skipUnchangedDigest = flag.Bool("skip-unchanged-digest", false, "If set to true, will only push images where the digest has changed.")
+	retryCount          = flag.Int("retry-count", 0, "Amount of times the push will be retried.")
```

Comment:
Should we validate this value? What happens when the retry count is `-1`? Some users may assume this to mean infinite retries, when in fact the container image would not even attempt to be pushed once.

Reply:
Good point. Do you think it's better to log using `log.Fatalln` if it's negative or set it to `0` and log a warning instead?